### PR TITLE
This is Colin Pattullo's mistake (Period 1)

### PR DESCRIPTION
**Pull request by ColinPat**
@twang This is Colin Pattullo's mistake from Period 1. I figured out what I did wrong, and fixed the problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# unit-1-work-in-progress
\ No newline at end of file
+# unit-1-work-in-progress
+
+public class Spikey{
+    public static void main(String args[]){
+    System.out.println(" \\/");
+    System.out.println(" \\\\//");
+    System.out.println("  \\\\\///");
```
Comment (katie03) [inline README.md]: You need to add a extra "\" character because it is an escape character and delete a few spaces to have it formatted correctly.

Comment (katie03) [inline README.md]: In line 5, you need to add an extra space before \\/. In lines 7 and 8, you should delete the spaces before the characters to have correct formatting.

Comment (katie03) [inline README.md]: You need to add a \ character because it is an escape character and is an illegal escape sequence otherwise.

Comment (katie03) [reply README.md]: In line 5, you need to add an extra space before \\\\/. In lines 7 and 8, you should delete the spaces before the characters to have correct formatting.

**Comment by twang** [comment]
Thanks @ColinPat and @katie03!

**Pull request closed without merging**